docs/topics/installing-cli-tool.adoc

```diff
@@ -42,6 +42,9 @@ To install using the downloadable `.zip` file:
 +
 When you encounter `<{ProductShortName}_HOME>` in this guide, replace it with the actual path to your {ProductShortName} installation.
 
+. Copy or save the `mta-cli` binary to a known directory.
++
+Continue to link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/7.0/html/cli_guide/installing_and_running_the_cli#cli-run_cli-guide[Running the CLI] to run the CLI command.
```

Comment:
Why are you using a link and not an xref? 

It is a mistake to use a hard link when an xref is an available option.

Reply:
Replaces Link with xref.
Preview shows that the xref sends to published MTA document.